clash_royale_service.py

```diff
@@ -33,12 +34,43 @@ def compare_card_levels(p1, p2):
     return compare_result
 
 
-class ClanMembersRanker:
+def compute_war_active(fame_hist: list) -> bool:
+    war_active = True
+    for week in fame_hist:
+        if week is None:
+            continue
+        elif week < 1200:

```

Comment:
Plz put this 1200 threshold in the schema.py and copy paste the comment explaining where this value comes from

Reply:
Done! MIN_FAME_WEEK